security/pkg/stsservice/tokenmanager/google/tokenexchangeplugin.go

```diff
@@ -504,13 +504,21 @@ func (p *Plugin) GetMetadata(forCA bool, xdsAuthProvider, token string) (map[str
 	}
 	gcpProjectNumber := p.GetGcpProjectNumber()
 	if !forCA && xdsAuthProvider == GCPAuthProvider && len(gcpProjectNumber) > 0 {
+		pluginLog.Debugf("forCA is false and gcp provider; returns a %v token", security.XDSBearerTokenPrefix)
```

Comment:
This seems excessively spammy. I don't think we need it per request?

Reply:
Removed.